README.md

```diff
@@ -12,7 +12,7 @@
 
 <div align="center">
 
-> [![Version badge](https://img.shields.io/badge/version-0.1.2-silver.svg)]()
+> [![Version badge](https://img.shields.io/badge/version-0.1.0-silver.svg)]()
```

Comment:
## pt

Realizar o comando `bumpversion patch` como descrito nas orientações de contribuição

## en

Make the `bumpversion patch` command as described on the contribution guidelines